source/conf.py

```diff
@@ -209,17 +209,22 @@
 suppress_warnings = ['epub.unknown_project_files']
 
 # -- Options for HTML output ---------------------------------------------------
-
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'default'
+
+from better import better_theme_path
+html_theme_path = [better_theme_path]
+html_theme = 'better'
+#html_theme = 'default'
 #html_theme = 'basic'
 #html_style = 'rsyslog.css'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-#html_theme_options = {}
+html_theme_options = {
+'inlinecss': ' @media (max-width: 820px) { div.sphinxsidebar { visibility: hidden; } }',
```

Comment:
I think this may be related to the resolution check that I mentioned in an earlier reply. I hadn't yet looked over the diff against the `conf.py` file yet when I responded.

It may not matter here, but for reference here are the resolution specs on the two devices I mentioned as my test cases earlier:

- LG V20 resolution is 2,560 x 1,440
- Nexus 7 (2013) resolution is 1920 x 1200

Reply:
that max width is from the better theme. Actually, we fix what seems to be a bug inside the theme here. maybe @alorbach can elaborate a bit more on that.

Reply:
physical resolution doesn't match browser resolution when browsing with mobile devices (unless you force desktop mode in chrome on android). So even if the mobile devices would have 4k resolutions, the design would be broken. 

Anyway the theme is kind of broken once the design breaks down to tablet / phone mode. fonts are layed over another, we workaround that problem by hidding the sphinxsidebar. 

To fix this we would have to fix the theme. And then we can also use the current theme (which has little bit better design in my opinion) and make it responsive.

Reply:
I think currently the theme renders unreadable on small screens. The new theme renders 90% OK. Not great, but much better readable. Nobody I know has time to write an optimized theme. My conclusion is that we should change as proposed and life with the fact that it is an incremental improvement, albeit not a perfect solution.

Reply:
@rgerhards @alorbach Going to respond in detail within the "main flow" of this issue instead of inline here.